routing_model/index.html

```diff
@@ -263,17 +270,11 @@ <h2>Deciding when to accept an incoming relocation</h2>
     If it is Some, we will relay the RPC to them instead of accepting the rellocation to our own section.
     </p>
   </div>
-  <button class="collapsible">PROCESSING_CANDIDATE</button>
+  <button class="collapsible">waiting_proofing</button>
   <div class="content">
     <p>We want to accept at most one incoming relocation at a time into our section.<br/>
-    The local variable: PROCESSING_CANDIDATE indicates whether someone is already passing the resource proof process, so we can't accept anyone else for now.<br/>
-    When PROCESSING_CANDIDATE is true and we reach consensus on ParsecExpectCandidate, we send a RefuseCandidate RPC to the would-be-incoming-node so they can try another section or try again later.
-    </p>
-  </div>
-  <button class="collapsible">AcceptAsCandidate</button>
-  <div class="content">
-    <p>If we are ready to accept an incoming node and we reach consensus on ParsecExpectCandidate, we will execute this flow to make them pass resource proof.<br/>
-    See AcceptAsCandidate diagram for details.
+    waiting_proofing function return the list of nodes that we have yet to resource proof, (States from WaitingCandidateInfo until it reach OnlineState).<br/>
+    When waiting_proofing() is not empty and we reach consensus on ParsecExpectCandidate, we send a RefuseCandidate RPC to the would-be-incoming-node so they can try another section or try again later.
```

Comment:
I would add an explanation for the leftmost branch here: if it's not empty, but it's the same guy, we resend our RelocateResponse.

Reply:
This behavior is not in this PR unless I messed up something?

Reply:
Oops. I was not looking at the exact correct graph...